### PR TITLE
Show `FM_BITCOIND_RPC` in a setup UI

### PR DESCRIPTION
**Pull request by dpc**
On top of #1289 

Screenshot from 2023-01-12 13-34-20

### CODE REVIEW

**Comment by dpc** [comment]
I wanted (and still want to) to add some tooltips or something like that give some helpful infomation, without cluttering the UI, but bootstrap (or rather popper.js failed me: the tooltips were off to the center, and gave up on figuring out why after a short struggle).

**Comment by dpc** [comment]
@justinmoon as it's mostly your code I'm messing with :)

**Comment by justinmoon** [comment]
will review this evening

**Comment by elsirion** [comment]
Re screenshot: p2p we don't speak websocket but a custom binary protocol over TLS, thus the schema should probably be `fedimint://…`

File: fedimintd/src/ui.rs [hunk 1]
```diff
@@ -214,10 +214,21 @@ async fn post_guardians(
 
 #[derive(Template)]
 #[template(path = "params.html")]
-struct UrlConnection {}
+struct UrlConnection {
+    ro_bitcoind_rpc: String,
```
Comment (elsirion) [inline fedimintd/src/ui.rs:218]: Can't this be an `Url`? Iirc that implements `Display`.

**Review by elsirion — APPROVED** [review]
I love how the re-arranged form fields look. It's becoming really professional!

File: fedimintd/src/ui.rs (hunk 1 above)
Comment (dpc) [reply fedimintd/src/ui.rs:218]: We don't always have `Url`, and it's `Option<Url>` and then askama's template seems too simplistic to work with it... 

Maybe there's a better way.

Comment (elsirion) [reply fedimintd/src/ui.rs:218]: You could have a `match` statement in askama iirc, but I agree that's probably not much better at that point (although you could hide other HTML elements like labels and headings depending on it).

**Pull request merged**